Bottom: 93753b11bf48e088b4a01490af3fabfa1a2fdf4a
Top:    95c506c15fe492aea8fe7df61a49b0dadb24de9c
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-03-06 14:59:09 +0000

Adjust STEP export to 0.2mm board thickness (Hack - need something quick!)


---

diff --git a/src/hid/common/object3d.c b/src/hid/common/object3d.c
index 82c89fa..191fd81 100644
--- a/src/hid/common/object3d.c
+++ b/src/hid/common/object3d.c
@@ -72,7 +72,7 @@
 #endif
 
 
-#define HACK_BOARD_THICKNESS MM_TO_COORD(2.3)
+#define HACK_BOARD_THICKNESS MM_TO_COORD(0.2)
 #define HACK_COPPER_THICKNESS MM_TO_COORD(0.035)
 #define HACK_PLATED_BARREL_THICKNESS MM_TO_COORD(0.08)
 #define HACK_MASK_THICKNESS MM_TO_COORD(0.01)
diff --git a/src/hid/step/step.c b/src/hid/step/step.c
index 1d725f3..d277acc 100644
--- a/src/hid/step/step.c
+++ b/src/hid/step/step.c
@@ -45,7 +45,7 @@
 
 #define CRASH fprintf(stderr, "HID error: pcb called unimplemented STEP function %s.\n", __FUNCTION__); abort()
 
-#define HACK_BOARD_THICKNESS MM_TO_COORD(2.3)
+#define HACK_BOARD_THICKNESS MM_TO_COORD(0.2)
 
 
 HID step_hid;
